src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -25,7 +26,8 @@
      */
     public AddCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap =
-                ArgumentTokenizer.tokenize(args, PREFIX_TAG_NAME, PREFIX_FILE_ADDRESS, PREFIX_TAG);
+                ArgumentTokenizer.tokenize(args, PREFIX_TAG_NAME, PREFIX_FILE_ADDRESS,
+                        PREFIX_OLD_TAG_NAME, PREFIX_COMMAND);
```

Comment:
Does add command parser need prefix_old_tag_name? Isn't the command tag f/filepath t/tagname?